app/src/main/java/es/wolfi/passman/API/Core.java

```diff
@@ -83,31 +86,104 @@ public static void setAPIPassword(String password) {
         Core.password = password;
     }
 
-    public static void requestAPIGET(Context c, String endpoint, final FutureCallback<String> callback) {
+    /**
+     * Old method (can do login after updating credentials with wrong values)
+     *
+     * @param c
+     * @param endpoint
+     * @param callback
+     */
+    public static void _requestAPIGET(Context c, String endpoint, final FutureCallback<String> callback) {
```

Comment:
Why keep it if there is an improved version?

Reply:
Wanted to test it more. Done.